packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -415,23 +443,26 @@ class HttpRouteSpec extends RouteSpec {
   }
 
   public bind(scope: Construct): RouteSpecConfig {
-    const prefixPath = this.match ? this.match.prefixPath : '/';
-    if (prefixPath[0] != '/') {
-      throw new Error(`Prefix Path must start with \'/\', got: ${prefixPath}`);
-    }
+    const pathMatchConfig = this.match?.path?.bind(scope);
 
+    // Set prefix path match to '/' if none of path matches are defined.
+    const prefixPathMatch = pathMatchConfig ? pathMatchConfig.prefixPathMatch : '/';
```

Comment:
No need for this complexity here - you can just do:

```suggestion
    const pathMatchConfig = (this.match?.path ?? HttRoutePathMatch.startsWith('/')).bind(scope);
```

And then below:

```ts
        prefix: pathMatchConfig.prefixPathMatch,
        path: pathMatchConfig.wholePathMatch,
```

Reply:
This is great know. Thanks!